src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/InteropServices/PlatformCompatibilityAnalyzer.cs

```diff
@@ -1074,17 +1058,16 @@ private static ISymbol GetEventAccessor(IEventSymbol iEvent, IOperation operatio
 
                     if (TryGetOrCreatePlatformAttributes(containingSymbol, true, platformSpecificMembers, out var callSiteAttributes))
                     {
-                        if (IsNotSuppressedByCallSite(operationAttributes, callSiteAttributes, msBuildPlatforms, out var notSuppressedAttributes))
+                        if (callSiteAttributes.Callsite != Callsite.Empty &&
```

Comment:
This check is being repeated in both if and else if parts. Consider moving the check to outer if, i.e.

```csharp
if (TryGetOrCreatePlatformAttributes(containingSymbol, true, platformSpecificMembers, out var callSiteAttributes) &&
    callSiteAttributes.Callsite != Callsite.Empty)
```

Reply:
Good point, those are not exactly equivalent, else part should only work when there were no platform attributes found in call-site, but the `callSiteAttributes.Callsite != Callsite.Empty` in else part is redundant, leftover from my previous tries